src/Components/Endpoints/src/FormMapping/WellKnownConverters.cs

```diff
@@ -37,7 +42,13 @@ static WellKnownConverters()
             { typeof(DateTimeOffset), new ParsableConverter<DateTimeOffset>() },
             { typeof(TimeSpan), new ParsableConverter<TimeSpan>() },
             { typeof(TimeOnly), new ParsableConverter<TimeOnly>() },
-            { typeof(Guid), new ParsableConverter<Guid>() }
+            { typeof(Guid), new ParsableConverter<Guid>() },
+            { typeof(IFormFileCollection), new FileConverter<IFormFileCollection>() },
+            { typeof(IFormFile), new FileConverter<IFormFile>() },
+            { typeof(IReadOnlyList<IFormFile>), new FileConverter<IReadOnlyList<IFormFile>>() },
+#if COMPONENTS
+            { typeof(IBrowserFile), new FileConverter<IBrowserFile>() }
```

Comment:
nit: Can we also do `IReadOnlyList<IBrowserFile>`? Otherwise it can't be used on RCLs if they want to support wasm

Reply:
Sure